1-mandatory/1-writers.js

```diff
@@ -62,7 +65,11 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  writers.forEach(writer => {
+    if(writer.age >= 40 && writer.age <= 49 && !writer.alive){
+      console.log(`Writer ${writer.firstName} ${writer.lastName} died at ${writer.age} years old.`)
+    }
+  })
 }
```

Comment:
Comment: Well done

Consider: Here is another way to do it. 
function logAllWriters() {
  // write your code to log all writers here
  for (let i = 0; i < writers.length; i++) {
    const writer = writers[i];
    console.log(`Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.`);
  }
}

Reply:
Thank you @softacoder. But here is condition about the age